public/src/components/channelManagement/campaigns/CampaignsEditor.tsx

```diff
@@ -147,12 +184,65 @@ function CampaignsEditor({ campaign }: CampaignsEditorProps): React.ReactElement
         updatePage={updatePage}
       />
       <div className={classes.scrollableContainer}>
-        <div className={classes.formContainer}>
-          <div className={classes.notesContainer}>
-            <div className={classes.notesHeader}>Notes (will be an editable RTE field):</div>
-            {description && <Typography>{description}</Typography>}
+        {name !== unassignedCampaign.name && (
+          <div className={classes.formContainer}>
+            <div className={classes.notesContainer}>
+              <div className={classes.notesHeader}>Campaign metadata and notes:</div>
+
+              <TextField
+                inputRef={register()}
+                error={errors.description !== undefined}
+                helperText={errors.description ? errors.description.message : ''}
+                onBlur={handleSubmit(updateDescription)}
+                name="description"
+                label="Description"
+                margin="normal"
+                variant="outlined"
+                disabled={!editMode}
+                fullWidth
+              />
+
+              <FormControlLabel
+                control={
+                  <Switch
```

Comment:
There's a really annoying bug with the switch where, on first click, it changes the `isActive` status of the campaign in the sidebar listing, but the appearance of the switch itself doesn't change. Clicking again on the switch will update its appearance to bring it in line with the campaigns new status. 

Suggestions on how to fix very welcome as this is a poor UX.

Reply:
Fixed